index.html

```diff
@@ -73,7 +73,8 @@
         "permissions-policy", "hr-time", "picture-in-picture", "mediacapture-streams"
       ]
     },
-    preProcess: [experimentalPreProcess]
+    preProcess: [experimentalPreProcess],
+    implementationReportURI: "https://wpt.fyi/results/compute-pressure?label=master&label=experimental&aligned"
```

Comment:
```suggestion
    implementationReportURI: "https://wpt.fyi/results/compute-pressure"
```

This shorter URL redirects to the latest.